crates/sargon/src/gateway_api/models/logic/response/transaction/status/transaction_status.rs

```diff
@@ -0,0 +1,36 @@
+use crate::prelude::*;
+
+#[cfg(test)]
+mod tests {
```

Comment:
Why is this test here and nit near the `TransactionStatusResponse` declaration?

Reply:
note that this is located under `logic/response/transaction/status`. If you go to the analogue folder under `types/response/transaction/status` you will also find a `transaction_status.rs` that holds the `TransactionStatusResponse` type.

I assume your confusion comes from the fact that the files are named `transaction_status.rs` and not `transaction_status_response.rs`.  I could rename both to the latter, although we skip the `_response` suffix for other similar cases (such as `TransactionSubmitResponse`), which I think makes sense since the root folder is `response/`. 

Let me know what you guys prefer, and we can follow such standard cc @CyonAlexRDX

Reply:
I think Ghenadie meant that these unit tests are placed in a separate file. And might instead be placed next to the declaration of the type..?

Reply:
oh! in such case I followed the `type` vs `logic` separation that we already have.

Reply:
I think that is fine